google_symptoms/delphi_google_symptoms/run.py

```diff
@@ -96,7 +93,7 @@ def run_module(params, logger=None):
             df = df.reset_index()
             sensor_name = "_".join([smoother, "search"])
             if len(df) == 0:
-                logger.info("No data for %s_%s_%s", geo_res, metric.lower(), sensor_name)
+                logger.info("No data for signal", signal=f"{geo_res}_{metric.lower()}_{sensor_name}")
```

Comment:
Match the above?
```suggestion
                logger.info("No data for signal", geo_res=geo_res, metric=metric.lower(), sensor_name=sensor_name)
```

Reply:
I wasn't sure since the original log wanted to capture it as a signal and not into seperate metric, sensor, geo....

Reply:
Yea, these old pipelines like to mix together geo resolution, metric and sensor name into a single name and call it signal, though that doesn't match our definition of signal anywhere else. Seems better to me to separate these out, since then you can do granular search in the logs without doing substring search.

Reply:
Two lines below this is a call to `create_export_csv()` [which combines metric and sensor in the output file name](https://github.com/cmu-delphi/covidcast-indicators/blob/2184da727a4a19acf53bcce7d6a14f1d7452d27c/_delphi_utils_python/delphi_utils/export.py#L110), in what will ultimately be the signal name.  How about:
```suggestion
                logger.info("No data for signal", geo_type=geo_res, signal=f"{metric}_{sensor_name}")
```



Should we use this opportunity to add better logging to `create_export_csv()`?  @aysim319, you have something planned to change doctor visits and hospital admissions to use this method, right?  Perhaps we can add the logging then.

Reply:
yeah.. I think it's better to revisit once all the indicators are using create_export_csv... because at least for hospital-admissions it doesn't quite use the metric and the sensor combination if I recall